src/main/resources/data/htm/lang/de_de.json

```
@@ -20,15 +20,15 @@
   "text.htm.trust": "§a%s §bwird vertraut",
   "text.htm.untrust": "§a%s §b wird nicht vertraut",
   "text.htm.global": " §a§lGlobal",
-  "text.htm.transfer": "§bBehälter wurde nach §a%s transferiert",
+  "text.htm.transfer": "§bBehälter wurde §a%s übertragen",
   "text.htm.unlocked": "§bBehälter entsperrt",
   "text.htm.key": "§b§lSchlüssel: §r§a%s",
   "text.htm.set": "§bDer Behälter wurde auf folgendes gesetzt: §r§a%s",
   "text.htm.key_set": "§bDer Schlüssel wurde geändert auf: §r§a%s",
   "text.htm.override": "§bDieser Behälter gehört: §r§a%s §cWird ignoriert",
   "text.htm.set_flag": "§r§a%s §bwurde auf §r§a%s §bgesetzt",
   "text.htm.flag": "§b§l%s: §r%s §r",
-  "text.htm.persist": "§bFortbestehen wurde umgeschaltet ",
+  "text.htm.persist": "§bFortbestehen: ",
```

Fortbestehen ist ein... komisches Wort, ich hab aber keine bessere Idee.
https://www.duden.de/synonyme/fortbestehen ?

---

Vielleicht solte man es tatsächlich am Besten einfach mit "Persistenz" übersetzen. Das Wort gibt es ja immerhin (https://www.duden.de/rechtschreibung/Persistenz) und trifft am Besten den eigentlichen Sinn.

---

Persistenz ist noch mehr ein Fremd/Fachwort als Fortbestehen; der Duden sagt ja es wird vorallem in Medizin und Biologie benutzt.
Man könnte vielleicht paraphrasieren und nicht ein Nomen benutzen / beschreiben, was die Option macht.

---

Prinzipiell ja, allerdings ist das hier ein wenig schwierig, da erst am Ende an den Text angehängt wird, ob die Option aktiviert oder deaktiviert ist. Darüber hinaus fällt mir keine wirklich taugliche Umschreibung ein. "Fortbestehen" ist vermutlich am leichtesten zu verstehen, klingt aber irgendwie nicht so gut.

---

> da erst am Ende an den Text angehängt wird
Das ist ein Problem das noch im code behoben werden muss

Falls es nur ein Wort sein muss, währe ich persöhnlich für "langlebig"

---

Da mir aktuell auch nichts besseres einfällt, würde ich das jetzt einfach mal vorschlagen.
```suggestion
  "text.htm.persist": "§bLanglebig: ",
```